parm/config/config.base.emc.dyn

```diff
@@ -227,11 +233,66 @@ export lobsdiag_forenkf=".true."
 export DO_GLDAS="YES"
 export gldas_cyc=00
 
-# run wave component
+# APP settings
+export APP="ATM"
+
+# Defaults:
 export DO_WAVE="NO"
+export DO_MOM6="NO"
+export DO_CICE="NO"
+export DO_GOCART="NO"
+export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_ugwpv1"
+
+if [ $APP = "ATMW" ]; then
+  export DO_WAVE="YES"
+elif [ $APP = "S2S" ]; then
+  export DO_MOM6="YES"
+  export DO_CICE="YES"
+  export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_coupled_nsstNoahmpUGWPv1"
+elif [ $APP = "S2SW" ]; then
+  export DO_WAVE="YES"
+  export DO_MOM6="YES"
+  export DO_CICE="YES"
+  export CCPP_SUITE="FV3_GFS_v16,FV3_GFS_v16_coupled_nsstNoahmpUGWPv1"
+elif [ $APP = "ATMAERO" ]; then
+  export DO_GOCART="YES"
+fi
+
 if [[ "$SDATE" -lt "2019020100" ]]; then DO_WAVE="NO" ; fi # no rtofs in GDA
 export WAVE_CDUMP="both"
 
+# Coupling settings
+
+export cpl=".true."
+export cplflx=".true."
+export cplice=".false."
+export cplgocart=".false."
+export cplwav=".false."
+export cplwav2atm=".false."
+
+if [ $DO_CICE = "YES" ]; then
+  export cplice=".true."
+fi
+if [ $DO_GOCART = "YES" ]; then
+  export cplgocart=".true."
+fi
+if [ $DO_WAVE = "YES" ]; then
+  export cplwav=".true."
+  export cplwav2atm=".true."
+fi
+
+# Temporarily override settings for aerosol coupling
+if [ $cplgocart = ".true." ]; then
+  export cplflx=".false."
+  export cplice=".false."
+  export cplocn=".false."
+  export cplwav=".false."
+  export cplwav2atm=".false."
+fi
+
+export nems_temp="cpld_wave"
+export nems_temp_cold="atm"
```

Comment:
Hmm... aren't these variables that kind of depend on the app? And then this is also something that could tell you if cplwav2atm=true/false.

Reply:
These are new to me so I need you and @WalterKolczynski-NOAA to tell me what/where they should be. Thanks!

Reply:
nems_temp_cold is only used in jobs/JGLOBAL_FORECAST_MEDCOLD which is not a job that is used anymore and probably shouldn't be included when we merge things into develop. Therefore I'd recommend just not adding this variable.  

for nems_temp here is what it should be defined on based on the $APP: 

case "${APP}" in
   ATM)
    nems_temp='atm'
  ATMW) 
     nems_temp='leapfrog_atm_wav'
   S2S)
     nems_temp='cpld'
   S2SW)
     nems_temp='cpld_wave'
   ATMAERO)
         nems_temp='atm_aer'